tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.py

```diff
@@ -0,0 +1,150 @@
+CVE_DB_PATH = os.path.join(WAZUH_PATH, 'queue', 'vulnerabilities', 'cve.db')
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_feed_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'data', 'feeds')
+configurations_path = os.path.join(test_data_path, 'wazuh_min_full_scan_interval.yaml')
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+min_full_scan_interval_values = [120]
+min_full_scan_interval_units = ['s', 'm', 'h', 'd']
+parameters = []
+metadata = []
+ids = []
+
+# Offline feeds
+buster_oval_feed_path = os.path.join(test_feed_path, vd.CUSTOM_DEBIAN_OVAL_FEED)
+debian_json_feed_path = os.path.join(test_feed_path, vd.CUSTOM_DEBIAN_JSON_FEED)
+nvd_json_feed_path = os.path.join(test_feed_path, vd.CUSTOM_NVD_FEED)
+
+for value in min_full_scan_interval_values:
+    for unit in min_full_scan_interval_units:
+        parameters.append({'MIN_FULL_SCAN_INTERVAL': f'{value}{unit}',
+                           'BUSTER_FEED_PATH': buster_oval_feed_path,
+                           'DEBIAN_JSON_FEED_PATH': debian_json_feed_path,
+                           'NVD_JSON_FEED_PATH': nvd_json_feed_path})
+        metadata.append({'min_full_scan_interval': f'{value}{unit}'})
+        ids.append(f'{value}{unit}')
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+
+# fixtures
+@pytest.fixture(scope='module', params=configurations, ids=ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope="module")
+def get_local_internal_options():
+    """Get configurations from the module."""
+    return local_internal_options
+
+
+@pytest.fixture(scope="function")
+def add_simulated_agent(get_configuration):
+    """Add a dummy agent, inserts in its database (sys_programs table) a test package,
+    and configures its database to appear to be up to date (sync_info table)."""
+    agent = ags.Agent(manager_address="localhost", os="debian10")
+    agent.set_module_status('syscollector', 'enabled')
+    agent.set_module_status('fim', 'disabled')
+    sender, injector = ags.connect(agent)
+    # Add the changes to the agent's database.
+    query_wdb(f"agent {agent.id} sql INSERT INTO sys_programs (scan_id,scan_time,format,name,priority,section,size,"
+              f"vendor,install_time,version,architecture,multiarch,source,description,location,triaged,cpe,msu_name,"
+              f"checksum,item_id) VALUES(0,'2021/04/07 22:00:00','deb','test package','optional','utils','7490',"
+              f"'Wazuh wazuh@wazuh.com',NULL,'1.0.0','amd64',NULL,NULL,'Test package',NULL,0,NULL,NULL,"
+              f"'e7dbc9bba5a0ee252866536225b952d3de7ea5cb','777fef8cc434b597769d102361af718d29ef72c1')")
+    query_wdb(f"agent {agent.id} sql UPDATE sync_info SET last_attempt = 1, last_completion = 1 where component = "
+              f"'syscollector-packages'")
```

Comment:
This could be another helper function... update sys_info or something like that.

Reply:
Done in: https://github.com/wazuh/wazuh-qa/pull/1369/commits/5045a8250c712c9081ddc6565b62c4c87b998a5c